drivers/overlay/ov_network.go

```diff
@@ -298,6 +298,14 @@ func (n *network) initSubnetSandbox(s *subnet) error {
 	return nil
 }
 
+func (n *network) cleanupStaleSandboxes() {
+	for i := 1; i < n.initEpoch; i++ {
+		path := osl.GenerateKey(fmt.Sprintf("%d-", i) + n.id)
```

Comment:
Given you are already using one Sprintf, why not using it for inserting the network id into the string ? This way you avoid another string buffer operation.

`
path := osl.GenerateKey(fmt.Sprintf("%d-%s", i, n.id))
`